open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml

```diff
@@ -27,6 +27,10 @@
         <node.downloadRoot>https://nodejs.org/dist/</node.downloadRoot>
         <npm.version>6.4.1</npm.version>
         <npm.downloadRoot>https://registry.npmjs.org/npm/-/</npm.downloadRoot>
+        <frontend.plugin.version>1.6</frontend.plugin.version>
```

Comment:
This is a regression. Changes were made in this area by e08e0086482ca9a868fcbd8e6a71f67c7ae770fe (7 Nov 2019). None of these versions should be here,